### PR TITLE
subtraction feature added

### PR DESCRIPTION
**Pull request by milanbhandari047**
---
name: Pull Request
about: Create a pull request to contribute to this repository
title: ''
labels: ''
assignees: ''
---

**Is your pull request related to an issue?**

If so, please link the issue here: [Link to the issue]

**Describe your Pull Request**

A clear and concise description of what you have done and why.

**Changes Made**

A brief summary of the changes you made.

**Screenshots**

If applicable, add screenshots to showcase the changes.

**Checklist**

- [ ] I have tested my changes thoroughly.
- [ ] My code follows the project's coding guidelines.
- [ ] I have added necessary documentation (if applicable).
- [ ] I have updated the README (if applicable).
- [ ] All new and existing tests passed.
- [ ] I have signed the Contributor License Agreement.

**Additional Comments**

Any additional information or context.

**Related Issues**

- Fixes #[Issue Number]
- Closes #[Issue Number]

### CODE REVIEW

File: cpp/addition.cpp [hunk 1]
```diff
@@ -7,11 +7,11 @@ int main() {
     cout << "Enter two numbers: ";
     cin >> num1 >> num2;
 
-    // Step 2: Performing Addition
+    // Step 2: Performing subtraction
```
Comment (rahuldahal) [inline cpp/addition.cpp:10]: The comment says *subtraction*, but the code performs *addition*.

File: cpp/addition
Comment (rahuldahal) [inline cpp/addition:1]: delete these binary files(files generated after compilation). Only commit the files that have the source code and not the compiled binary.

File: cpp/multip;ication.cpp
Comment (rahuldahal) [inline cpp/multip;ication.cpp:1]: **Rename** the file name correctly. Don't rush my friend, you have plenty of time.

File: cpp/subtraction
Comment (rahuldahal) [inline cpp/subtraction:1]: Remove this binary file as well

**Review by rahuldahal — CHANGES_REQUESTED** [review]
I have commented on the files and changes are expected.

File: multiplication.cpp
Comment (rahuldahal) [inline multiplication.cpp:1]: Remove this! It is already on cpp/multiplication.cpp

File: cpp/subtraction
Comment (rahuldahal) [inline cpp/subtraction:1]: Asked you to delete this and addition!

File: cpp/addition
Comment (rahuldahal) [inline cpp/addition:1]: Asked you to delete this and subtraction!

**Review by rahuldahal — CHANGES_REQUESTED** [review]

**Review by rahuldahal — CHANGES_REQUESTED** [review]
remove the `.vscode` folder from the commit.
dot folders/files might be hidden in the explorer.
From the command line do the following:
```bash
git rm -r .vscode
git commit -m "removed .vscode settings"
git push -u origin main
```
Then delete this PR and create a new one.

**Comment by rahuldahal** [comment]
@milanbhandari047 Don't be frustrated because of all the changes I've been requesting. If I were you, I would see this as a new learning opportunity. Happy coding 🥂

**Comment by milanbhandari047** [comment]
> remove the `.vscode` folder from the commit. dot folders/files might be hidden in the explorer. From the command line do the following:
> 
> ```shell
> git rm -r .vscode
> git commit -m "removed .vscode settings"
> git push -u origin main
> ```
> 
> Then delete this PR and create a new one.

@rahuldahal  can you help me how to delete the PR?

**Pull request closed without merging**